contrib/redhat/keyszer.spec

```diff
@@ -0,0 +1,87 @@
+# Fedora Linux
+%global git_date 20220715
+%global git_rev_full 4ddf5fffb2f4330dcb535c8b4651616351c371f6
+%global git_rev %(c=%{git_rev_full}; echo ${c:0:6})
+%global keyszer_user keymapper
+
+Name:		keyszer
+Version:	0.6.0
+Release:	1.%{git_date}git%{git_rev}%{?dist}
+License:	GPLv3
+Summary:	X keyboard remappring tool
+URL:		https://github.com/joshgoebel/%{name}
+#Source0:	https://github.com/joshgoebel/{name}/archive/{name}-{version}.tar.gz
```

Comment:
I'm not familiar but it looks like we're using git rev vs the release version?  Is there a reason to prefer that for this package?

Reply:
This is standard packaging versioning for [intermediate versions](https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/) (post-release git snapshots)

Reply:
Ah... I guess that's my fault... starting with 0.7 though I'd probably rather have people build releases against release versions... so i'd prefer the git option be commented out (for people to uncomment if they want), but default to releases.

Reply:
As `master` is not release branch I propose ti get this form in it.
This means 'this spec is working with this commit OK, tune on your taste".
But before release we can replace git things with simply version.
So stable release will be shipping with its own stable spec.

Reply:
But if you confused with strange source name this is [rule](https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_commit_revision)

Reply:
> so i'd prefer the git option be commented out (for people to uncomment if they want), but default to releases.

Fixed